azure-devops/ecommerce/06_pagopa-ecommerce-transactions-functions.tf

```diff
@@ -0,0 +1,127 @@
+variable "pagopa-ecommerce-transactions-functions" {
+  default = {
+    repository = {
+      organization    = "pagopa"
+      name            = "pagopa-ecommerce-transactions-functions"
+      branch_name     = "refs/heads/main"
+      pipelines_path  = ".devops"
+      yml_prefix_name = null
+    }
+    pipeline = {
+      enable_code_review = true
+      enable_deploy      = true
+      sonarcloud = {
+        # TODO azure devops terraform provider does not support SonarCloud service endpoint
+        service_connection = "SONARCLOUD-SERVICE-CONN"
+        org                = "pagopa"
+        project_key        = "pagopa_pagopa-ecommerce-transactions-functions"
+        project_name       = "pagopa-ecommerce-transactions-functions"
+      }
+    }
+  }
+}
+
+locals {
+  # global vars
+  pagopa-ecommerce-transactions-functions-variables = {
+    cache_version_id = "v1"
+    default_branch   = var.pagopa-ecommerce-transactions-functions.repository.branch_name
+  }
+  # global secrets
+  pagopa-ecommerce-transactions-functions-variables_secret = {
+
+  }
+  # code_review vars
+  pagopa-ecommerce-transactions-functions-variables_code_review = {
+    danger_github_api_token = "skip"
+    sonarcloud_service_conn = var.pagopa-ecommerce-transactions-functions.pipeline.sonarcloud.service_connection
+    sonarcloud_org          = var.pagopa-ecommerce-transactions-functions.pipeline.sonarcloud.org
+    sonarcloud_project_key  = var.pagopa-ecommerce-transactions-functions.pipeline.sonarcloud.project_key
+    sonarcloud_project_name = var.pagopa-ecommerce-transactions-functions.pipeline.sonarcloud.project_name
+  }
+  # code_review secrets
+  pagopa-ecommerce-transactions-functions-variables_secret_code_review = {
+
+  }
+  # deploy vars
+  pagopa-ecommerce-transactions-functions-variables_deploy = {
+    github_connection = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_github_rw_name
+
+    # acr section
+    k8s_image_repository_name           = replace(var.pagopa-ecommerce-transactions-functions.repository.name, "-", "")
```

Comment:
this is an azure function and will not be deployed on aks, so these variables are not needed

Reply:
Fixed in https://github.com/pagopa/pagopa-azure-devops/pull/155/commits/433981e89f67b83daa48583a61db7ffb4226fbd9